src/olympia/reviewers/forms.py

```diff
@@ -299,7 +302,25 @@ def create_option(self, *args, **kwargs):
         return option
 
 
-class ReviewForm(forms.Form):
+class WithAttachmentMixin:
```

Comment:
Can you implement this as a [validator](https://docs.djangoproject.com/en/4.2/ref/validators/#writing-validators) instead?  (named `validate_review_attachment` or similar)